### PR TITLE
Feature/republish live records 3

### PR DESCRIPTION
**Pull request by josephlewisnz**
Added the ability to autopublish items that are already live 
Same request  #289  for 3 Branch

### CODE REVIEW

File: src/GridFieldOrderableRows.php
Comment (dhensby) [inline src/GridFieldOrderableRows.php]: This phpdoc is for a parameter that doesn't exist.

**Review by dhensby — CHANGES_REQUESTED** [review]
This looks good, thanks.

It would be great to have a test to cover this functionality too

File: src/GridFieldOrderableRows.php
Comment (josephlewisnz) [reply src/GridFieldOrderableRows.php]: Removed that phpdoc  param

**Comment by josephlewisnz** [comment]
Regarding tests, I'm not the most experienced in that space. I could try taking a stab at it sometime late next week.

**Comment by purplespider** [comment]
Can we please get this merged, @dhensby?

People have literally been trying for years to get it merged into the correct branch and a release: #289, #339, #340 😮‍💨 And for those of us pointing composer to the specific commit it's become an upgrade blocker.

Thank you! 🤞🏻

**Comment by dhensby** [comment]
I don't have merge permissions on the symbiote repos anymore - I can't merge this, I'm afraid

**Comment by purplespider** [comment]
@sabina-talipova Are you able to assist with merging this?

**Pull request merged**
